pkg/pool-manager/pool_test.go

```diff
@@ -413,5 +414,53 @@ var _ = Describe("Pool", func() {
 			Expect(newPod.Annotations[networksAnnotation]).To(Equal(afterAllocationAnnotation[networksAnnotation]))
 		})
 	})
+	Describe("Multus Network Annotations API Tests", func() {
+		It("should allocate requested mac-address with single ip-address request in network annotation", func() {
+			poolManager := createPoolManager("02:00:00:00:00:00", "02:00:00:00:00:01", &vmConfigMap)
+			Expect(poolManager).ToNot(Equal(nil), "failed to create pool-manager")
```

Comment:
The expectation string should be what **should** happen. So in this case "Should create pool-manager". If we align to this rule everywhere in the code, then we can just read the "strings" in the tests to understand what is going on.

Reply:
For me it is more clear to see these kind of error messages in after test suite logs,
specifying what failed and why instead of what should have happen.
I will align with this kind of error messages :).

Reply:
Hopefully, you will read the code more often than seeing it fail. The positive string helps you read the code alone.

Here is an example :

```go
			It("should return a Result with RequeueAfter set (trigger re-reconciliation)", func() {
				result, err := reconciler.Reconcile(request)
				Expect(err).ToNot(HaveOccurred(), "Should succcessfully reconcile the request")
				Expect(result).To(Equal(reconcile.Result{RequeueAfter: 666 * time.Second}), "Result should have expected requeue value")
			})
```

The result (written in red is):

```
  Result should have expected requeue value
  Expected
      <reconcile.Result>: {
          Requeue: false,
          RequeueAfter: 5000000000,
      }
  to equal
      <reconcile.Result>: {
          Requeue: false,
          RequeueAfter: 666000000000,
      }
```

So you see the expectation and the actual result.

The biggest benefit of this is, that the code is way more readable when you just follow strings and don't need to revert their meaning to see what **should** happen.

Reply:
Thanks for explaining I will change it :)